roles/openshift_router/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+# TODO: Centralized openshift_config openshift_facts??
+# TODO: anchor this on a master
+- name: Query for OpenShift router
+  command: openshift ex router --credentials={{ openshift_config }}
```

Comment:
We already set the variable both on the node and the master, it would be nice if the router and registry create commands could query the master for the template string from the config and use that when generating the pod config.